examples/cmake/X3D_ADD_EXAMPLE.cmake

```diff
@@ -0,0 +1,9 @@
+macro(AddExample example_dir case list_files)
+  install(DIRECTORY DESTINATION ${example_dir}/${case})
+  set(local_list "")
+  list(APPEND local_list ${list_files})
+  foreach(ff IN LISTS local_list)
+    install(FILES ${ff} DESTINATION ${example_dir}/${case})
+  endforeach()
```

Comment:
Could you not just `foreach` over `list_files` instead of building a local copy?

Reply:
From memory `list_files` is not a list since it is passed as an argument of the macro, therefore i need this trick of creating a local variable and force it to be a list.